Code/Framework/AzCore/AzCore/EBus/Event.h

```diff
@@ -58,6 +58,12 @@ namespace AZ
 
         Event& operator=(Event&& rhs);
 
+        //! Take the handlers registered with the other event
+        //! and move them to this event. The other will event
+        //! will be cleared after call
+        //! @param other event to move handlers
+        Event& StealHandlers(Event&& other);
```

Comment:
Nitpick: "Steal" sounds so sinister. How about "ClaimHandlers"?

Reply:
Yes, I will change the name to sound less malicious